internal/plugins/helm/v1/init.go

```diff
@@ -128,7 +129,22 @@ func (p *initPlugin) InjectConfig(c *config.Config) {
 
 // Run will call the plugin actions
 func (p *initPlugin) Run() error {
-	return cmdutil.Run(p)
+	if err := cmdutil.Run(p); err != nil {
+		return err
+	}
+
+	// Assume projectName was validated by Validate().
+	wd, err := os.Getwd()
+	if err != nil {
+		return fmt.Errorf("error getting the current path: %v", err)
+	}
+	projectName := strings.ToLower(filepath.Base(wd))
+	// Run the scorecard "phase 2" plugin.
+	if err := scorecard.RunInit(projectName); err != nil {
+		return err
+	}
+
+	return nil
```

Comment:
IMO it is a post-action and should be in `PostScaffold()` instead.

Reply:
This is definitely a scaffolding thing since its writing more files into the project. I disagree with moving it to `PostScaffold()`

Reply:
I guess you could classify all phase 2 plugin actions as post-scaffolds. In my mind this is part of running the plugin, not cleaning up or validating the scaffolded files. It depends on how you perceive plugin execution.